src/pages/components/form/usage.mdx

```diff
@@ -117,215 +115,378 @@ do you gain by collecting this information?
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| _Enabled_   | When a text input is live but a user is not directly interacting with it. This is commonly referred to as the default or normal state of the component. An enabled text input field can contain no content, placeholder text, or user generate content.           |
+| _Active_    | When a user is actively typing content into the field.                                                                                                                                                                                                            |
+| _Focus_     | When a user `tabs` to or clicks on the text input, it becomes focused, indicating the user has successfully navigated to the component.                                                                                                                           |
+| _Error_     | When the user input is invalid or a required text input has not been filled in. It can also be triggered due to a system error. This state requires a user response before data can be submitted or saved.                                                        |
+| _Warning_   | When you need to call the user's attention to an exception condition. The condition might not be an error but can cause problems if not resolved.                                                                                                                 |
+| _Disabled_  | When the user is not allowed to interact with the text input due to either permissions, dependencies, or pre-requisites. The disabled state completely removes the interactive function from a component. The styling is not subject to WCAG contrast compliance. |
+| _Skeleton_  | Use on an initial page load to indicate that the text input has not yet fully loaded.                                                                                                                                                                             |
+| _Read-only_ | _Coming soon!_                                                                                                                                                                                                                                                    |
+
 <Row>
-<Column colLg={8}>
+<Column colLg={12}>
 
-![Example of product configuration properties using the required pattern](images/form-usage-required.png)
+<Tabs>
+
+<Tab label="Fixed">
+
+![Examples of fixed input states](images/form-usage-states-fixed.png)
```

Comment:
The skeleton state input field needs to bottom align with the disabled field to the left of it in the fixed image.
<img width="829" alt="Screen Shot 2023-01-24 at 2 32 48 PM" src="https://user-images.githubusercontent.com/43969356/214390668-a3bf3063-fb0d-4a60-a8a2-e53f8e2e8615.png">